docs/roadmap.md

```diff
@@ -0,0 +1,40 @@
+# Roadmap
+
+## Heapster v0.19
+
+### Timeline
+October 2015
+
+### Features
+- Support for InfluxDB v0.9
+- Support for Reimann
+
+## Heapster v0.20
+
+### Timeline
+December 2015
+
+### Features
+- Support for application metrics.
+- Improving existing model API
+  - Support label based queries in the APIs
+- Internal optimizations for performance and stability
+- Scalability - support <= 250 node clusters
```

Comment:
Maybe just "support 250 node clusters" instead of \<= so people know we will actually support 250.

Same for \<= 1000 later.